Documentation/contributing.rst

```diff
@@ -37,19 +37,25 @@ contribute to Cilium:
+| `Docker-Compose <https://docs.docker.com/compose/install/>`_                     | OS-Dependent             | N/A (OS-specific)                                                             |
 +----------------------------------------------------------------------------------+--------------------------+-------------------------------------------------------------------------------+
-+ `Cmake  <https://cmake.org/download/>`_                                          | OS-Dependent             | N/A (OS-specific)                                                             |
+| `Cmake  <https://cmake.org/download/>`_                                          | OS-Dependent             | N/A (OS-specific)                                                             |
 +----------------------------------------------------------------------------------+--------------------------+-------------------------------------------------------------------------------+
-+ `Bazel <https://docs.bazel.build/versions/master/install.html>`_                 | 0.13.0                   | N/A (OS-specific)                                                             |
+| `Bazel <https://docs.bazel.build/versions/master/install.html>`_                 | 0.13.0                   | N/A (OS-specific)                                                             |
 +----------------------------------------------------------------------------------+--------------------------+-------------------------------------------------------------------------------+
-+ `Libtool <https://www.gnu.org/software/libtool/>`_                               | >= 1.4.2                 | N/A (OS-specific)                                                             |
+| `Libtool <https://www.gnu.org/software/libtool/>`_                               | >= 1.4.2                 | N/A (OS-specific)                                                             |
 +----------------------------------------------------------------------------------+--------------------------+-------------------------------------------------------------------------------+
-+ `Automake <https://www.gnu.org/software/automake/>`_                             | OS-Dependent             | N/A (OS-specific)                                                             |
+| `Automake <https://www.gnu.org/software/automake/>`_                             | OS-Dependent             | N/A (OS-specific)                                                             |
 +----------------------------------------------------------------------------------+--------------------------+-------------------------------------------------------------------------------+
-+ `Kubecfg <https://github.com/ksonnet/kubecfg>`_                                  | >=0.8.0                  | go get github.com/ksonnet/kubecfg                                             |
+
+The following Golang dependencies can be installed by running `make install-cilium-golang-deps`:
```

Comment:
We should explain when this is used, and so when to install this. Alternatively, you can make the rule `install-cilium-test-deps` (this isn't a go dependency, it's used as a tool).